src/main/java/com/google/api/codegen/transformer/py/PythonSurfaceNamer.java

```diff
@@ -453,4 +453,27 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
         throw new IllegalStateException("Invalid development status");
     }
   }
+
+  @Override
+  public String getPrintSpec(String spec) {
+    // Escaers don't work here. They only map from characters to strings.
```

Comment:
"Escapers"? I assume they're previously existing classes. Maybe name the package so it's clearer what you're referring to.

Reply:
Done